keyboards/romac/info.json

```
@@ -0,0 +1,13 @@
+{
+  "keyboard_name": "RoMac",
+  "url": "",
+  "maintainer": "qmk",
+  "width": 3,
+  "height": 4,
+  "layouts": {
+    "LAYOUT_ortho_3x4": {
```

This should be changed to:

```suggestion
    "LAYOUT": {
```

See the review on `romac.h`.